core/include/config.h

```diff
@@ -66,4 +66,12 @@ struct config_t {
 
 #define HAX_MAX_VCPUS 16
 
+#ifdef __NetBSD__
```

Comment:
nit: `core/` code should use `HAX_PLATFORM_*` which is more readable:

```suggestion
#ifdef HAX_PLATFORM_NETBSD
```